backends/ref/ceed-ref-operator.c

```diff
@@ -1168,13 +1168,13 @@ int CeedOperatorCreateFDMElementInverse_Ref(CeedOperator op,
   ierr = CeedVectorGetArrayRead(q_weight, CEED_MEM_HOST, &q_weight_array);
   CeedChkBackend(ierr);
   ierr = CeedCalloc(num_elem, &elem_avg); CeedChkBackend(ierr);
+  const CeedScalar qf_value_bound = max_norm*100*CEED_EPSILON;
```

Comment:
What about this relative value as the cutoff in both places @jedbrown? In practice, due to the perturbation, I don't know that we actually get fdm diagonal values that are below that threshold now.

Reply:
I think this relative bound would be fine, perhaps with size-dependent defaults. For example, SciPy uses `max(m,n) * eps`: https://github.com/scipy/scipy/blob/v1.7.0/scipy/linalg/basic.py#L1330-L1337

Reply:
In our case that would be something like `sqrt(elem_size)*CEED_EPSILON`

Reply:
Looks good to me.